app/code/core/Mage/Api/Model/Cron.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Api
+ * @copyright  Copyright (c) 2023 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+class Mage_Api_Model_Cron
+{
+    /**
+     * Clean session table
+     *
+     * @param Mage_Cron_Model_Schedule $schedule
+     * @return $this
+     */
+    public function sessionCleanUp($schedule)
```

Comment:
```suggestion
    public function cleanOldSessions($schedule)
```
I'd use the cleanOldSessions which is the same name used in app/code/core/Mage/Api/Model/Resource/User.php

but also, is it ok for this method to have an argument? I never say any cron method to have arguments, I can't even test it directly because obviously `Mage::getModel('api/cron')->sessionCleanUp()` triggers an error

Reply:
You can test like this: `Mage::getModel('api/cron')->cleanOldSessions(null);`

Reply:
but the $schedule parameter is not used in the method itself, why is it there if it's not used?

Reply:
I prefer it because it is called with the param `$schedule`, line 298 and line 320:
https://github.com/OpenMage/magento-lts/blob/0f032f375598b50d1c9c553206a9c750796bf1d6/app/code/core/Mage/Cron/Model/Observer.php#L298-L320

I usually set a message `$schedule->setMessages($msg);` which will be saved in the table.

Reply:
See also https://github.com/OpenMage/magento-lts/blob/0f032f375598b50d1c9c553206a9c750796bf1d6/app/code/core/Mage/Reports/Model/Observer.php#L13-L21

`$schedule` is not used.

Reply:
Actually, it's customary in the core to have the param `$schedule`, see [app\code\core\Mage\Sales\Model\Observer.php](https://github.com/OpenMage/magento-lts/blob/main/app/code/core/Mage/Sales/Model/Observer.php)